kitsune/community/jinja2/community/metrics.html

```diff
@@ -16,7 +16,8 @@
         <div class="inline-controls">
           <div>
             <select id="toggle-cohort-type">
-              <option value="contributor:kb:en-US">{{ _('en-US') }}</option>
+              <!-- The locale code is not localizable - the data are for English only. -->
```

Comment:
The backend jinja template comment should be something like
``{# Some Comment #}``